pkg/karmadactl/deinit.go

```diff
@@ -61,7 +61,7 @@ func NewCmdDeInit(parentCommand string) *cobra.Command {
 		},
 	}
 
-	flags := cmd.PersistentFlags()
```

Comment:
I can't tell the exact difference between `PersistentFlags` and `Flags`, can you remind me @carlory ?

Reply:
The `PersistentFlags` will be passed to all `subcommands` of the current command. We have imported `karmadactl option` command, it seems to capture all `PersistentFlags`, so we can't use `PersistentFlags` in specific subcommand.

Reply:
Do you mean `PersistentFlags` `used` to add the flags that are intended used by `subcommands`, but the flags can't be shown in the usage of the current command?

Take here, for example, we added following `--namespace`/`--kubeconfig`/`--context` to the `deinit` as `PersistentFlags`, so these flags can not be shown.  right?

Reply:
@RainbowMango Yes，reference [https://stackoverflow.com/questions/63495992/what-is-the-difference-between-go-cobra-persistentflags-and-flags](url)

Reply:
Great thanks @helen-frank. I got it.